lib/src/lib.rs

```diff
@@ -4,16 +4,17 @@
 #[cfg(feature = "colors")]
 use colored::Colorize;
 
+// Re-exports.
+// TODO: are these types really necessary to be exposed? Should we wrap them in our own?
```

Comment:
Should we remove this`TODO`comment and remind us to handle this question through a _enhancement_ ticket like [this one](https://github.com/polyphene/kythera/issues/45)?

Reply:
yup, makes sense Philippe, opened https://github.com/polyphene/kythera/issues/59. And removed the todo. Thanks